samples/subsys/lorawan/class_a/sample.yaml

```diff
@@ -6,3 +6,4 @@ sample:
 tests:
    samples.lorawan.class_a:
      platform_allow: 96b_wistrio
+     platform_allow: rak4631_nrf52840
```

Comment:
Please don't add until you get lorawan working

Reply:
yes you're right, I forgot it